ppr-ui/src/components/mhrTransportPermit/LocationChange.vue

```diff
@@ -30,9 +30,49 @@
           >
             <template #item="{ props, item }">
               <v-list-item
-                v-bind="props"
-                @click="handleLocationTypeChange(item.props.value)"
-              />
+                v-bind="isListItemDisabled(item.props.value) ? null : props"
+                :title="null"
+                :ripple="isListItemDisabled(item.props.value) ? false : true"
+                :class="{ 'disabled-item' : isListItemDisabled(item.props.value) }"
+                @click="isListItemDisabled(item.props.value) ? null : handleLocationTypeChange(item.props.value)"
+              >
+                <v-tooltip
+                  v-if="isListItemDisabled(item.props.value)"
+                  location="right"
+                  contentClass="left-tooltip"
+                  transition="fade-transition"
+                >
+                  <template #activator="{ props }">
+                    <div v-bind="props">
```

Comment:
Just a heads up, this may* limit the click area.

I think we had a related issue before where clicking outside the list-item-title, on the very edges of the row, might trigger Vuetifys prompt to close but may not trigger the bound event. 

Just a small thing to be aware of or verify!

Reply:
Good call, thanks; even though this item is not clickable, I removed the div wrapper.